mypy/test/data.py

```diff
@@ -67,6 +67,7 @@ def parse_test_case(case: DataDrivenTestCase) -> None:
     files: list[tuple[str, str]] = []  # path and contents
     output_files: list[tuple[str, str | Pattern[str]]] = []  # output path and contents
     output: list[str] = []  # Regular output errors
+    output_inline_start: int  # Start index of output inline assertions
```

Comment:
Don't think you need to declare this up front. The other variables are declared here so we can say what should be in the list/dict.

Reply:
66d8305e2db69e24fc9566aea13e39c514ce1b20